.github/workflows/python-package.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python package
```

Comment:
we don't build any package, can you rename it and the filename too ?

Reply:
true. fixed.